src/sass/base/_clipboard.scss

```diff
@@ -0,0 +1,6 @@
+// Hide temporary textareas used for copying into the clipboard
+body {
+    > textarea {
+        visibility: hidden;
+    }
+}
```

Comment:
Well, this works, but I meant setting inline CSS with JavaScript :slightly_smiling_face: 

```javascript
element.style.display = 'none';
```

Adding a global CSS rule with its own CSS file is a bit overkill in my opinion and - while a top-level textarea is highly unlikely - might have side effects.

Reply:
Sigh...

Reply:
Yes, but less files in less locations = higher cohesion and easier maintenance :slightly_smiling_face:

Reply:
It depends on our global CSS anyway unless you want to prevent rendering it... which could (and probably will) be optimised away and ignored by the browser.